docs/source/_static/custom.css

```
@@ -1,3 +1,24 @@
 .section dl.py:not(:first-of-type) {
     margin-top: 1.5em;
 }
+
+.strike {
+    text-decoration: line-through;
+}
+
+.strong {
+    font-weight: bold;
+}
+
+.em {
+    font-style: italic;
+}
+
+.red {
+    color: #dd0000;
```

Why not `#f00`? That's the de facto (mIRC) standard for "Red" (color 04).

```suggestion
    color: #f00;
```